product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -112,7 +112,8 @@ Now that your endpoint service is created, you can connect it to the cluster VPC
  In your application's AWS account, select **VPC** and then select **Endpoints**. Select the endpoint you created previously and use the DNS name provided in the details section to access your cluster.
 
     ```shell     
-    psql -h vpce-XXXXXXXXXXXXXXXXXXXX.eu-west-1.vpce.amazonaws.com -U edb_admin      
+    # Replace the port 5432 below if that has been changed in the Big Animal DB Configuration
```

Comment:
```suggestion
    # Update the port number if your BigAnimal database configuration 
    # uses a different port  
```

Reply:
@amacvar if you agree with my suggestion, I'll commit it and merge the PR.

Reply:
Actually we need the `-p 5432` in the psql string else with an EPAS client it defaults to port 5444 while BA default is 5432.

Reply:
My suggestion doesn't remove the `p 5432` in the psql string. I'm going to commit it. I think the comment is an improvement and I think we can merge it.